source/simulator/core.cc

```diff
@@ -549,8 +549,8 @@ namespace aspect
   {
     // first produce some output for the screen to show where we are
     {
-      const char *unit = (parameters.convert_to_years) ? "years" : "seconds";
-      const double factor = (parameters.convert_to_years) ? 1./year_in_seconds : 1.0;
+      const char *unit = (parameters.convert_to_years ? "years" : "seconds");
+      const double factor = (parameters.convert_to_years ? 1./year_in_seconds : 1.0);
 
       pcout << "*** Timestep " << timestep_number
             << ":  t=" << time *factor << ' ' << unit
```

Comment:
Huh, I had put a space to the right of the asterisk here:
```suggestion
            << ":  t=" << time * factor << ' ' << unit
```
Bizarrely, the indentation script removes it again. Same for the following line. I have no idea why...

Reply:
Interesting. astyle must be confused here.

Reply:
do you want to try ``(time * factor)`` ?

Reply:
Yes, that works.